app/assets/javascripts/modules/track-smart-answer.js

```diff
@@ -3,35 +3,40 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 (function (Modules) {
   'use strict'
 
-  Modules.TrackSmartAnswer = function () {
-    this.start = function (element) {
-      var nodeType = element.data('smart-answer-node-type')
-      var flowSlug = element.data('smart-answer-slug')
+  function TrackSmartAnswer (element) {
+    this.$module = element
+  }
 
-      if ((nodeType === undefined) || (flowSlug === undefined)) {
-        return
-      }
+  TrackSmartAnswer.prototype.init = function () {
```

Comment:
With the change of initialiser we could probably simplify the rest of this module quite significantly by being a bit tactical.

```
TrackSmartAnswer.prototype.init = function () {
  if (!this.flowSlug || !GOVUK.analytics || !GOVUK.analytics.trackEvent) return
  
  if (this.nodeType === 'outcome') {
    GOVUK.analytics.trackEvent('Simple Smart Answer', 'Completed', {
      label: this.flowSlug,
      nonInteraction: true,
      page: window.location.pathname      
    })
  }
}
```

Reply:
I think setting `page: window.location.pathname` like this will break the tests as one of them relies on being able to mock the path.

Also, why does the suggested guard clause stop checking if `nodeType` is null? 👇 
`if (!this.flowSlug || !GOVUK.analytics || !GOVUK.analytics.trackEvent) return`

To be honest the brief in the card was to just remove jQuery, I think completely restructuring the code might be a tad out of scope.

Reply:
> I think setting `page: window.location.pathname` like this will break the tests as one of them relies on being able to mock the path.

Ah yes, that makes sense. I did wonder why such a trivial method existed when there was so much other complexity.

> 
> Also, why does the suggested guard clause stop checking if `nodeType` is null? 👇
> `if (!this.flowSlug || !GOVUK.analytics || !GOVUK.analytics.trackEvent) return`

it doesn't need to check if that is null because there's an explicit check that it matches 'outcome' (`if (this.nodeType === 'outcome')` in my code, and the [very unusual 1 outcome switch](https://github.com/alphagov/frontend/blob/546e11b8bd71484bc77cd13f0ab2489ce169f3b1/app/assets/javascripts/modules/track-smart-answer.js#L30-L35) in the existing 

> To be honest the brief in the card was to just remove jQuery, I think completely restructuring the code might be a tad out of scope.

I'd consider basic simplifications in scope, it seems unnecessary to leave the WTFs around for a future person to ponder.